mesonbuild/mtest.py

```diff
@@ -1422,8 +1423,12 @@ def __init__(self, test: TestSerialisation, env: T.Dict[str, str], name: str,
         else:
             timeout = self.test.timeout * self.options.timeout_multiplier
 
+        if self.options.quiet:
+            verbose = 0
+        else:
+            verbose = max(test.verbose, self.options.verbose)
```

Comment:
I think `test.verbose` is a bool, not an int?

Reply:
It was changed to int to support specifying more than one verbose flag.